packages/button/__tests__/Button.test.tsx

```diff
@@ -1,8 +1,14 @@
 import { render, screen } from '@testing-library/react'
+import { axe } from 'jest-axe'
 
 import type { ButtonProps } from '../src/'
 import Button from '../src/'
 
+it('is accessible', async () => {
+  const { container } = render(<Button>text</Button>)
+  expect(await axe(container)).toHaveNoViolations()
+})
+
```

Comment:
Assuming we will add this test to every suite going forward?

Reply:
Yes!